setup.py

```diff
@@ -39,6 +41,10 @@ def read_version():
 else:  # <= Python 3.4
     tests_requires += ['matplotlib<3.0', 'Pillow<6.0']
 
+# pytorch: python 2.7 require future
+if sys.version_info < (3, 0):
+    tests_requires += ['future']
```

Comment:
Could you please elaborate on why future is needed?

Reply:
python2.7 import error.
` No module named builtins`
for avoid this, Install future.

see.
https://discuss.pytorch.org/t/building-pytorch-from-source-in-conda-fails-in-pytorch-caffe2-operators-crash-op-cc/42859/3
https://github.com/hyperopt/hyperopt/issues/273

```bash
root@1f2b2ea83553:~# pip install torch
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Collecting torch
  Downloading torch-1.4.0-cp27-cp27mu-manylinux1_x86_64.whl (753.4 MB)
     |████████████████████████████████| 753.4 MB 2.5 kB/s
Installing collected packages: torch
Successfully installed torch-1.4.0
root@1f2b2ea83553:~# python
Python 2.7.17 (default, Feb 26 2020, 17:18:08)
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/torch/__init__.py", line 19, in <module>
    from ._six import string_classes as _string_classes
  File "/usr/local/lib/python2.7/site-packages/torch/_six.py", line 23, in <module>
    import builtins
ImportError: No module named builtins
```